.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. State why this change is required and list any dependencies that are required for this change.
```

Comment:
@Mastersam07 plz comment it out... not needed unnecessarily

Reply:
Okay 👍